exasol_integration_test_docker_environment/test/test_test_environment_nameservers.py

```diff
@@ -18,39 +19,26 @@ def setUpClass(cls):
                 utils.INTEGRATION_TEST_DOCKER_ENVIRONMENT_DEFAULT_BIN,
                 clean_images_at_close=False)
 
-    def setUp(self):
-        self.client = docker.from_env()
-
     @classmethod
     def tearDownClass(cls):
-        try:
-            cls.test_environment.close()
-        except Exception as e:
-            print(e)
+        close_environments(cls.test_environment)
 
     def tearDown(self):
-        try:
-            self.on_host_docker_environment.close()
-        except Exception as e:
-            print(e)
-        try:
-            self.google_cloud_docker_environment.close()
-        except Exception as e:
-            print(e)
-        self.client.close()
+        close_environments(self.on_host_docker_environment, self.google_cloud_docker_environment)
```

Comment:
I think it is better, if we don't store the environment in self and make the cleanup in the test method itself

Reply:
Done